### ISSUE TITLE
Timestamp in result-oriented remediation script header is wrong

### ISSUE THREAD

**Issue by jan-cerny** [OpenSCAP/openscap#772]
I generate a bash script from ARF results using this command:

```
oscap xccdf generate fix --result-id xccdf_org.open-scap_testresult_xccdf_org.ssgproject.content_profile_common arf.xml > fix.sh
```

Evaluation start time in the script header is wrong. It should report the time when the evaluation started. Instead, it shows the time when the fix was generated from an ARF.

See:

```
# Evaluation Start Time:  2017-07-04T11:34:14
# Evaluation End Time:  2017-07-04T11:26:07
```

It says that evaluation was started after it was finished 😬

**Comment by mpreisler** [comment]
Fixed by #804

**Comment by jan-cerny** [comment]
:1st_place_medal:
